src/test/java/utils/AssertUtils.java

```diff
@@ -0,0 +1,23 @@
+package utils;
+
+import org.hamcrest.Matcher;
+import org.junit.Assert;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class AssertUtils {
+
+    private static final String ASSERT_MESSAGE = "Assert that: {}{} [Actual value:] {}{}[Expected value:] {}";
+
+    public static <T> void assertThat(String message, T actual, Matcher<T> expected) {
+        log.info(ASSERT_MESSAGE, message, System.lineSeparator(), actual, System.lineSeparator(), expected);
+        Assert.assertThat(message, actual, expected);
+    }
+
+    public static void assertTrue(String message, boolean condition) {
+        log.info(ASSERT_MESSAGE, message, System.lineSeparator(), condition, System.lineSeparator(), Boolean.TRUE);
+        Assert.assertTrue(message, condition);
+    }
```

Comment:
could you please add a Java doc for both methods for anyone to be clear what your methods are doing and how it's working